extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -93,7 +104,61 @@ class InstrumentationService {
           }
         });
       }
+    } else if (eventType === AnalyticsEventType.CLICK) {
+      this.ensureClickListener_();
+      this.clickObservable_.add(
+          this.createSelectiveListener_(listener, opt_selector));
```

Comment:
An unspecified opt_selector indicates the trigger will fire on any click event (regardless of target).